dynamo/preprocessing/gene_selection.py

```diff
@@ -317,13 +319,13 @@ def get_vaild_CM(
     if total_szfactor is not None and total_szfactor in adata.obs.keys():
         szfactors = adata.obs[total_szfactor].values[:, None] if total_szfactor in adata.obs.columns else None
 
-    if szfactors is not None and relative_expr:
-        if issparse(CM):
-            from sklearn.utils import sparsefuncs
-
-            sparsefuncs.inplace_row_scale(CM, 1 / szfactors)
-        else:
-            CM /= szfactors
+    # if szfactors is not None and relative_expr:
+    #     if issparse(CM):
+    #         from sklearn.utils import sparsefuncs
+    #
+    #         sparsefuncs.inplace_row_scale(CM, 1 / szfactors)
+    #     else:
+    #         CM /= szfactors
```

Comment:
remove the code here. and you may move this to the utils and keep it as a util function?

Reply:
In my opinion, this is a well-written function that is heavily dependent on the gene selection procedure, and it may be better to keep it in this file instead of moving it to a util file. Additionally, since we already have a relative_expr parameter that determines whether we need to divide gene expression values, we can simply set the parameter to False to avoid the division here.

Reply:
`valid_CM, detected_bool = get_vaild_CM(adata, layer, relative_expr=False, **SVRs_kwargs)`